qa/rpc-tests/pegging.py

```diff
@@ -271,6 +289,23 @@ def write_bitcoin_conf(datadir, rpcport, rpcpass=None, p2p_port=None, connect_po
         if "confirmations" not in tx or tx["confirmations"] == 0:
             raise Exception("Peg-in confirmation has failed.")
 
+    print("Test pegout")
+    test_pegout(bitcoin.getnewaddress(), sidechain)
+
+    print("Test pegout Segwit (P2SH of witness script)")
+    parent_chain_addr = bitcoin.getnewaddress()
+    parent_chain_sw_addr = bitcoin.addwitnessaddress(parent_chain_addr)
+    test_pegout(parent_chain_sw_addr, sidechain)
+
+    print("Test pegout Garbage")
+    parent_chain_addr = "garbage"
+    try:
+        test_pegout(parent_chain_addr, sidechain)
```

Comment:
sorry, wasn't being clear enough here, I want the transaction to "pass", but for the output introspection of it to behave how we expect

Reply:
ie handcraft using rawtx api, then introspect